src/store/store.js

```diff
@@ -1,8 +1,11 @@
 /* @flow */
-import { createStore, applyMiddleware } from "redux";
+import { applyMiddleware, createStore } from "redux";
 import thunk from "redux-thunk";
+import Reactotron from "../../ReactotronConfig";
 import rootReducer from "../reducers/rootReducer";
 
 export default function configureStore() {
-  return createStore(rootReducer, applyMiddleware(thunk));
+  if (__DEV__) {
+    return Reactotron.createStore(rootReducer, applyMiddleware(thunk));
```

Comment:
Don’t import reactotron for prod, import it here using `require` instead of `import`